Source/WebKit/UIProcess/WebBackForwardList.cpp

```diff
@@ -549,15 +549,15 @@ String WebBackForwardList::loggingString()
     builder.append("WebBackForwardList 0x"_s, hex(reinterpret_cast<uintptr_t>(this)), " - "_s, m_entries.size(), " entries, has current index "_s, m_currentIndex ? "YES"_s : "NO"_s, " ("_s, m_currentIndex ? *m_currentIndex : 0, ')');
 
     for (size_t i = 0; i < m_entries.size(); ++i) {
-        const char* prefix;
+        ASCIILiteral prefix;
         if (m_currentIndex && *m_currentIndex == i)
-            prefix = " * ";
+            prefix = " * "_s;
         else
-            prefix = " - ";
+            prefix = " - "_s;
         builder.append('\n', prefix, m_entries[i]->loggingString());
     }
 
-    return makeString("\n", builder.toString());
+    return makeString('\n', builder.toString());
```

Comment:
Lets put the `'\n'` into the `StringBuilder` at the start of the function and avoid this expensive prepend operation.